examples/with-docker-multi-env/docker/staging/Dockerfile

```
@@ -21,14 +21,14 @@ WORKDIR /app
 COPY --from=deps /app/node_modules ./node_modules
 COPY . .
 # This will do the trick, use the corresponding env file for each environment.
-COPY .env.staging.sample .env.production
+COPY .env.staging.sample .env.staging
```

```suggestion
COPY .env.staging.sample .env.production
```
This also is correct as is since `.env.staging` won't be loaded by Next.js as it's not a recognized .env filename x-ref: https://github.com/vercel/next.js/blob/f93c685281feb9089597f2c23fd893411ad4a7f1/packages/next-env/index.ts#L102-L111